packages/validation/src/User/index.ts

```diff
@@ -65,31 +74,43 @@ export const createUserValidationSchema = Yup.object().shape({
     })
     .required('Please specify your birth date'),
   organisation: Yup.number().required(),
-  department: Yup.string().required(),
-  position: Yup.string().required(),
-  email: Yup.string().email(),
+  department: Yup.string()
+    .min(2)
+    .max(50)
+    .required(),
+  position: Yup.string()
+    .min(2)
+    .max(50)
+    .required(),
   telephone: Yup.string()
-    .min(2, 'telephone must be at least 2 characters')
-    .max(30, 'telephone must be at most 20 characters')
-    .matches(phoneRegExp, 'Phone number is not valid')
-    .required('telephone must be at least 2 characters'),
-  telephone_alt: Yup.string()
-    .min(2, 'telephone must be at least 2 characters')
-    .max(30, 'telephone must be at most 20 characters')
-    .matches(phoneRegExp, 'Phone number is not valid')
-    .notRequired(),
+    .min(2)
+    .max(30)
+    .matches(phoneRegExp, 'telephone number is not valid')
+    .required(),
+  telephone_alt: Yup.string().max(50),
```

Comment:
Isn't it better to use same check for telephone and telephone_alt but just make it as not required or as it was previously inside update schema?

Reply:
Well, I think we should be more flexible with telephone alt, for example, the default user officer user has such a telephone alt that fails the validation but looks valid number.
Phone number validation, in general, is very hard, as we only let people register who has ORCID account we shouldn't be very strict.
For regular phone numbers removing whitespace, dash, parentheses, and checking only for numbers should be also sufficient I think (e.g. I can't register with my own phone number unless I remove some formating although it's still a valid number.)

Reply:
Maybe for phone numbers we should use some special inputs like https://github.com/bl00mber/react-phone-input-2 or https://github.com/alexplumb/material-ui-phone-number where we can support formatting and easier validation.

Reply:
I personally wouldn't go that far, it's not really important and it's unlikely that a scientist would spam the phone number with something stupid in the user office app.